docs/aws/audit/route53monitoring/rules/config_global_resources.mdx

```diff
@@ -23,6 +23,219 @@ CBP
+3. In the AWS Config dashboard, select "Rules" from the left-hand navigation pane.
+4. In the Rules page, look for the rule named "route53-global-resource-record-sets". If this rule is present and enabled, AWS Config is checking for global resources in Route53. If the rule is not present or is disabled, AWS Config is not checking for global resources in Route53.
+</Accordion>
+
+<Accordion title='Using CLI'>
+1. Install and configure AWS CLI: Before you can start using AWS CLI, you need to install it on your local machine and configure it with your AWS account credentials. You can do this by running the following commands:
+
+   Installation:
+   ```
+   pip install awscli
+   ```
+   Configuration:
+   ```
+   aws configure
+   ```
+   You will be prompted to provide your AWS Access Key ID, Secret Access Key, Default region name, and Default output format.
+
+2. List AWS Config Rules: Once AWS CLI is set up, you can list all the AWS Config rules in your account by running the following command:
+   ```
+   aws configservice describe-config-rules
+   ```
+   This command will return a list of all the AWS Config rules in your account.
+
+3. Check for Global Resources Rule: Look for a rule with the name "route53-global-resource-record-sets". This rule checks whether AWS Config includes global resources in Route53. If this rule is not present, it means that AWS Config is not configured to include global resources in Route53.
+
+4. Check Rule Compliance: If the "route53-global-resource-record-sets" rule is present, you can check its compliance status by running the following command:
+   ```
+   aws configservice get-compliance-details-by-config-rule --config-rule-name route53-global-resource-record-sets
+   ```
+   This command will return the compliance status of the rule. If the status is "NON_COMPLIANT", it means that AWS Config is not including global resources in Route53.
+</Accordion>
+
+<Accordion title='Using Python'>
+1. Install the necessary AWS SDK for Python (Boto3) if you haven't done so already. You can install it using pip:
+
+```python
+pip install boto3
+```
+
+2. Import the necessary modules and create a session using your AWS credentials:
+
+```python
+import boto3
+
+session = boto3.Session(
+    aws_access_key_id='YOUR_ACCESS_KEY',
+    aws_secret_access_key='YOUR_SECRET_KEY',
+    region_name='YOUR_REGION'
+)
+```
+
+3. Use the AWS Config service client to describe the configuration recorder:
+
+```python
+config = session.client('config')
+
+response = config.describe_configuration_recorder_status(
+    ConfigurationRecorderNames=[
+        'default',
+    ]
+)
+
+print(response)
+```
+
+4. Check the 'recordingGroup' field in the response. If 'includeGlobalResourceTypes' is set to False, then global resources are not included in AWS Config:
+
+```python
+if not response['ConfigurationRecordersStatus'][0]['recordingGroup']['includeGlobalResourceTypes']:
```

Comment:
<b>Secret:</b> Secret Keyword <br/>
                    <b><a href="https://console.cloudanix.com/code?t=secrets" target='_blank'>Click here for more details</a></b>